MetaMask/metamask-extension
# Support for type 0 transaction

Fixes: #12189

Explanation:  

Example transaction type-0 submitted with the code changes: https://rinkeby.etherscan.io/tx/0x233988d13774032aa86b6514ed63934debe607388a1661e37be3bd81efcbd975

Manual testing steps:  
  - Goto [test dapp](https://metamask.github.io/test-dapp/) and create a legacy transaction on a network that supports EIP-1559.
  - The transaction should be visible as legacy transaction in metamask
  - Submit the transaction and it should show as type-0 transaction on etherscan
